README.md

```diff
@@ -181,9 +181,17 @@ Configuration format is described in detail in the
 * `failover_priority` (`list-of-string`): failover priority;
 * `roles` (`list-of-strings`, required if `replicaset_alias` specified): roles to be enabled on the replicaset;
 * `all_rw` (`boolean`, optional): indicates that that all servers in the replicaset should be read-write;
-* `weight` (`number`, optional): vshard replicaset weight (matters only if `vshard-storage` role is enabled.
+* `weight` (`number`, optional): vshard replicaset weight (matters only if `vshard-storage` role is enabled;
 * [DEPRECATED] `cartridge_failover` (`boolean`, optional): boolean flag that
   indicates if eventual failover should be enabled or disabled;
+* `data_dir` (`string`, optional, default: `/var/lib/tarantool/`):  a directory where application data files will be stored;
```

Comment:
Please, add a note that all these parameters wouldn't work with default RPM and DEB packages created by Cartridge CLI (since this packages contains unit files with default paths).

Reply:
A part of this options is used only with TGZ, please note about it too.